src/rp2_common/pico_stdio_uart/stdio_uart.c

```diff
@@ -76,6 +76,15 @@ void stdio_uart_init_full(struct uart_inst *uart, uint baud_rate, int tx_pin, in
     stdio_set_driver_enabled(&stdio_uart, true);
 }
 
+void stdio_uart_deinit() {
+    if (uart_instance != NULL)
+    {
+        stdio_set_driver_enabled(&stdio_uart, false);
+        uart_deinit(uart_instance);
```

Comment:
this needs a `uart_tx_wait_blocking(uart_instance);` here, otherwise we'll start loosing output (and wsl's minicom goes all wonky)